packages/shared/providers.yaml

```diff
@@ -6821,7 +6910,17 @@ yahoo:
     auth_mode: OAUTH2
     authorization_url: https://api.login.yahoo.com/oauth2/request_auth
     token_url: https://api.login.yahoo.com/oauth2/get_token
+    proxy:
+        base_url: https://${connectionConfig.apiBaseUrl}
```

Comment:
Could we always put `.yahooapis.com`?

Reply:
We have 2 apis for yahoo, Fantacy Sports API with `fantasysports.yahooapis.com` and `api.login.yahoo.com` for the `sign in with Yahoo` that uses the same oauth2 currently supported.

Reply:
can it be `apiDomain` then?

Reply:
we should probably have a way to tell customers that this is their responsibility not the user'

Reply:
I have updated this with the latest [commit](https://github.com/NangoHQ/nango/pull/3155/commits/5114e7cb33009b4b0ff2259cc8aec1ec949d3073)